client/a8c-for-agencies/sections/signup/agency-details-form/hooks/use-countries-and-states.ts

```diff
@@ -47,8 +47,8 @@ export interface Option {
 }
 
 export function useCountriesAndStates() {
-	const { data: countriesList } = useCountries();
 	const translate = useTranslate();
+	const { data: countriesList } = useCountries();
```

Comment:
Nit: This seems like unnecessary change, so it can be removed to reduce the noise.